gr-digital/grc/digital_gfsk_mod.block.yml

```diff
@@ -29,6 +29,13 @@ parameters:
     options: ['True', 'False']
     option_labels: ['On', 'Off']
     hide: ${ ('part' if str(log) == 'False' else 'none') }
+-   id: to_unpack
```

Comment:
Did `unpack` not work for `id`? Could use `do_unpack`.

Reply:
Im not sure. I didnt try. But even if it works, I feel uneasy using `unpack` even in the yaml because there's already a variable named `unpack` inside gfsk.py. Ive renamed all instances of `to_unpack` to `do_unpack`